app/src/main/java/com/cmput301f20t21/bookfriends/repositories/RequestRepository.java

```diff
@@ -46,8 +46,9 @@ public static IRequestRepository getInstance() {
                 .whereEqualTo("status", REQUEST_STATUS.BORROWED.toString()).get();
     }
 
-    public Task<List<Request>> getAllRequestsByUsername(String username) {
-        return requestCollection.whereEqualTo("requester", username).get()
+    public Task<List<Request>> getAllRequestsByUsername(String username, REQUEST_STATUS status) {
```

Comment:
maybe rename this to getRequestsByUsernameAndStatus or just getRequests? currently the arguments do not match the method name.